Concepts/concept_data_type.md

```diff
@@ -0,0 +1,67 @@
+# Concept Data Type
+
+## Overview
+
+* Concept Data type defines the structured format you desired the data to be represented as.
+
+* The current types are as follows:
+    
+    1. Numeric – any data represented numerically, also allows you to classify critical values and units, e.g. age, height, and liters consumed per day.
+    2. Coded – allows answers to be only those provided, e.g. Blood type can only be “A,” “B,” and “O”
+    3. Text – Open ended responses
+    4. N/A – the standard datatype for any non-query-like concepts, e.g. symptoms, diagnoses, findings, anatomy, misc, etc.
+    5. Document 
+    6. Date – structured day, month, and year
+    7. Time – structured time response
+    8. DateTime – structured response including both the date and the time
+    9. Boolean – checkbox response, e.g. yes or no queries
+    10. Rule 
+    11. Structured 
+
+
+## Available operations. 
+
+1. [List concept_data types](#list-concept-data-types)
+4. [Delete a concept_data type](#delete-a-concept_data-type)
+
+
+### List concept data types
+
+* #### List all non-retired concept data types.
+    
+    Quickly filter concept data types with a given search query.Returns a `404 Not Found` status if concept data type not exists. 
+    If user not logged in to perform this action,a `401 Unauthorized` status returned.
+    
+    ##### Query Parameters
+
+    Parameter | Type | Description
+    --- | --- | ---
+    *q* | `Search Query` | Display Name of concept data Type.
+
+    ```console
+    GET /conceptdatatype?q="Search Query"
```

Comment:
Does this work for you? In my testing, I get an empty list back if I try to search (i.e., any value for `q`).

Reply:
Hi @bmamlin executing without any q filter it works and gives a list. But the q filter is not working.

Reply:
Okay. So, let's not document a feature that isn't supported. Document the ability to get all concept data types, but don't include a search parameter in this case.

Reply:
sure will do